src/WebJobs.Extensions.DurableTask/IOrchestrationServiceFactory.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using DurableTask.Core;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask.Options;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask
+{
+    public interface IOrchestrationServiceFactory
```

Comment:
All public types and members need to be documented with XML doc comments. I believe StyleCop and/or the compiler should be flagging this. If it's too much work to do it now, let's open an issue to track adding these (or a generic issue to track fixing all warnings).